scripts/kubevirt/repo_groups.sql

```diff
@@ -141,14 +52,11 @@ set
 
 update gha_repos set alias = 'kubevirt/kubevirt' where name like '%kubevirt' or name = 'kubevirt/';
 
-insert into gha_repo_groups(id, name, alias, repo_group, org_id, org_login) select id, name, alias, coalesce(repo_group, name), org_id, org_login from gha_repos on conflict do nothing;
-insert into gha_repo_groups(id, name, alias, repo_group, org_id, org_login) select id, name, alias, org_login, org_id, org_login from gha_repos where org_id is not null and org_login is not null and trim(org_login) != '' on conflict do nothing;
-
 select
   repo_group,
   count(*) as number_of_repos
 from
-  gha_repo_groups
+  gha_repos
```

Comment:
This attemptsb to update to old `gha_repos` table - we are using `gha_repo_groups` instead since a long time, will close this PR.